latency/latency.go

```diff
@@ -0,0 +1,104 @@
+package latency
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const defaultAvgLatency = "0.58"
+const jsonFileLocation = "./data.json"
```

Comment:
Could you `embed` this data into the repo? That way we do not even need to put the downloader.

Reply:
Fixed